src/main/scala/gpp/ui/forms/EnumViewMultipleSelect.scala

```diff
@@ -0,0 +1,217 @@
+  onChange:             js.UndefOr[FormDropdown.OnChange] = js.undefined,
+  onChangeE:            js.UndefOr[FormDropdown.OnChangeE] = js.undefined,
+  onClose:              js.UndefOr[Callback] = js.undefined,
+  onCloseE:             js.UndefOr[OnClose] = js.undefined,
+  onFocus:              js.UndefOr[Callback] = js.undefined,
+  onFocusE:             js.UndefOr[OnFocus] = js.undefined,
+  onLabelClick:         js.UndefOr[Callback] = js.undefined,
+  onLabelClickE:        js.UndefOr[OnLabelClick] = js.undefined,
+  onMouseDown:          js.UndefOr[Callback] = js.undefined,
+  onMouseDownE:         js.UndefOr[OnMouseDown] = js.undefined,
+  onOpen:               js.UndefOr[Callback] = js.undefined,
+  onOpenE:              js.UndefOr[OnOpen] = js.undefined,
+  onSearchChange:       js.UndefOr[OnSearchChange] = js.undefined,
+  onSearchChangeE:      js.UndefOr[OnSearchChangeE] = js.undefined,
+  open:                 js.UndefOr[Boolean] = js.undefined,
+  openOnFocus:          js.UndefOr[Boolean] = js.undefined,
+  placeholder:          js.UndefOr[String] = js.undefined,
+  pointing:             js.UndefOr[Pointing] = js.undefined,
+  renderLabel:          js.UndefOr[RenderLabel] = js.undefined,
+  required:             js.UndefOr[Boolean] = js.undefined,
+  scrolling:            js.UndefOr[Boolean] = js.undefined,
+  search:               js.UndefOr[Boolean | SearchFunction] = js.undefined,
+  searchInput:          js.UndefOr[ShorthandS[VdomNode]] = js.undefined,
+  searchQuery:          js.UndefOr[String] = js.undefined,
+  selectOnBlur:         js.UndefOr[Boolean] = js.undefined,
+  selectOnNavigation:   js.UndefOr[Boolean] = js.undefined,
+  selectedLabel:        js.UndefOr[JsNumber | String] = js.undefined,
+  simple:               js.UndefOr[Boolean] = js.undefined,
+  tabIndex:             js.UndefOr[String | JsNumber] = js.undefined,
+  text:                 js.UndefOr[String] = js.undefined,
+  tpe:                  js.UndefOr[String] = js.undefined,
+  trigger:              js.UndefOr[VdomNode] = js.undefined,
+  upward:               js.UndefOr[Boolean] = js.undefined,
+  width:                js.UndefOr[SemanticWidth] = js.undefined,
+  wrapSelection:        js.UndefOr[Boolean] = js.undefined,
+  modifiers:            Seq[TagMod] = Seq.empty
+)(implicit
+  val enum:             Enumerated[A],
+  val show:             Show[A],
+  val effect:           Effect[F]
+) extends ReactProps[EnumViewMultipleSelect[Any, Any]](EnumViewMultipleSelect.component) {
+  def apply(mods: TagMod*): EnumViewMultipleSelect[F, A] = copy(modifiers = modifiers ++ mods)
+}
+
+object EnumViewMultipleSelect {
+  type Props[F[_], A] = EnumViewMultipleSelect[F, A]
+
+  protected val component =
+    ScalaComponent
+      .builder[Props[Any, Any]]
+      .stateless
+      .render_P { p =>
+        implicit val show   = p.show
+        implicit val effect = p.effect
+
+        FormSelect(
+          js.undefined,
```

Comment:
Why are these undefined. Would we want the user to have the choice to set them?

Reply:
Hmmm. I just copied these from the EnumViewSelect. I'll look into what they are.

Reply:
These are my omission. I should have added the parameter names. These are the properties that we don't want modified in an Enum select, for example: the ability to add new values. (I'm not 100% sure about `as`).

Reply:
I'm adding parameter names for those so it will be more clear why they aren't allowed to be specified.